src/utils/xrQSlim/src/MxPropSlim.cpp

```diff
@@ -342,7 +342,7 @@ unsigned int MxPropSlim::check_local_validity(unsigned int v, const float* vnew)
             mxv_cross3(n, f_n, d_yx); // n = ((y-x)^(v-x))^(y-x)
             mxv_unitize(n, 3);
 
-            // assert( mxv_dot(d_vx, n, 3) > -FEQ_EPS );
+            // R_ASSERT( mxv_dot(d_vx, n, 3) > -FEQ_EPS );
```

Comment:
Commented assertion? Maybe remove it at all?

Reply:
No, they mark potential problem places and can be useful when debugging.